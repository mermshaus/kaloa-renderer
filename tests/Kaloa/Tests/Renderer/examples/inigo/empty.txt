[][bogus]

[[url=http://example.org]link[/url]
