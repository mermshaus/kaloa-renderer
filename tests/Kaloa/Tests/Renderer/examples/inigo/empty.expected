<p>[][bogus]</p>

<p>[<a href="http://example.org" title="Open &quot;http://example.org&quot;">link</a></p>
